### ISSUE TITLE
[OJS] HTML syntax errors in HTML help

### ISSUE THREAD

**Issue by jprk** [pkp/pkp-lib#275]
Hello,

I just came across a small syntax error in HTML help templates in OJS 2.4.5. When I open `[journal_base]/help/view/editorial/topic/000044`
* there are two `<body>` tags in the displayed HTML
* the whole displayed list ends with `</ul>` which has not been opened anywhere

Code snippet:
```html
[...]
<body> <!-- fist body tag is ignored -->

<script type="text/javascript">
<!--
if (self.blur) { self.focus(); }
// -->
</script>


<div id="container"> <!-- this probably also comes from a template that includes the rest -->
<body id="body" class="pkp-common pkp-openJournalSystems">
[...]
</ul></div> <!-- this has not been opened anywhere -->
		</div>
[...]
```

It isn't something crucial, just to let you know that the templates probably need a bit of polishing.

-- jprk

**Comment by ctgraham** [comment]
In 2.4.5 stable, the duplicate body tags are here:
https://github.com/pkp/pkp-lib/blob/34f9c4797d94a36f09b09917e8158dcf67e581ed/templates/help/header.tpl#L58
vs.
https://github.com/pkp/pkp-lib/blob/34f9c4797d94a36f09b09917e8158dcf67e581ed/templates/help/header.tpl#L68
This has already been corrected in dev 2.4 with https://github.com/pkp/pkp-lib/commit/5e0a3498cb70f42c133380a32f08363cf919ac92

The mis-matched ol/ul is here in 2.4.5 (en_US):
https://github.com/pkp/ojs/blob/ojs-stable-2_4_5/help/en_US/editorial/topic/000044.xml#L19
vs.
https://github.com/pkp/ojs/blob/ojs-stable-2_4_5/help/en_US/editorial/topic/000044.xml#L60
It remains unaddressed in dev 2.4.

**Comment by asmecher** [comment]
Thanks for reporting, @jprk, and thanks for the additional fix, Clinton! Those fixes will both be included in OJS 2.4.6.
